### DISCUSSION TITLE
File > Send by email isn't working for me

### DISCUSSION THREAD

**Discussion by mikehd1959** [sumatrapdfreader/sumatrapdf#2501] [Q&A]
First time I've tried using the function and it simply won't launch my default email client in Windows 10 (Thunderbird). I just checked and Thunderbird is my default email app. I've looked in Options and Advance Options but can't see anything that mentions email or mail.

thanks

**Comment by kjk** [comment]
Oh well, it seems that the code we have no longer works in windows 10/11.

**Reply by GitHubRulesOK** [reply]
I must admit never bothering to test but blindly assumed that the majority of users like me will see 
![image](https://user-images.githubusercontent.com/12754913/164949455-74c1a1f0-e9de-442c-b235-210e711b9c03.png)

however to check if default Thunderbird works I can only go so far as to say without an account it attempts to send email 
![image](https://user-images.githubusercontent.com/12754913/164949665-948948bc-e5d0-4d17-b4aa-6e6f5f7294f2.png)

but cannot verify without an account if the attachment would be added

**Comment by jg-github** [comment]
My problem relates to the Epyrus email client. (http://www.epyrus.org/)

Thanks for your help.

![Sumatra-NoEmail](https://user-images.githubusercontent.com/3807863/215592833-660e3ee9-5fea-4edd-a386-81434fdbcceb.jpg)

**Reply by GitHubRulesOK** [reply]
MS have shifted a lot of underlying feature in the name of security, I used to be able to view remote pdf and other file types via Trident (the core of Internet Explore) Recently that seems ? (unconfirmed via other older devices) to have been fully removed with the EOL of Internet explorer.

File associations such as epub and pdf were messed about with the introduction of Edge.

Mailto may have depended on one or both aspects , however cant currently load either of above to test for present, perhaps reach out to https://forum.palemoon.org/viewforum.php?f=73 (For what its worth I can use Pale Moon with SumatraPDF to browse PDFs, but have not tried xps/djvu recently)

**Comment by jg-github** [comment]
OK, I've tried the Epyrus forum. Let's see if I get any responses. I haven't had any luck there recently getting replies to my various questions. 

I have the impression that after doing a really great job with the email client so far, the dev, athenian200, feels a bit overwhelmed in the attempt to go forward. 

I sincerely appreciate what's been done so far, and I've used Epyrus as my default mail client for about three weeks now.

In comparison to the overwhelming pleasure I've derived from Epyrus, my little problem in conjunction with SumatraPDF is a drop in the bucket.

**Comment by mikehd1959** [comment]
I am now able to send pdf via my email client, Thunderbird. No idea when it started working but I had totally forgot about this issue.

**Reply by jg-github** [reply]
> I am now able to send pdf via my email client, Thunderbird. No idea when it started working but I had totally forgot about this issue.

Thanks. This confirms my suspicion that it has to do with the inconsistent nomenclature in Epyrus.
